### DISCUSSION TITLE
World War mechanic

### DISCUSSION THREAD

**Discussion by 1ByteBit** [thatguycy/WorldDynamics-Engine#30] [Ideas]
If a certain amount of nations (for example, 90%) are at war, then a world war is initiated. When a world war is started, a number of things could happen, a couple examples being no cost on teleports but a higher cooldown, and lower costs for owning a nation in general.

There would have to be a config to determine all this so server owners could decide what specifically happens, such as the percentage required to go to war, any cost changes, etc etc.

**Comment by stoleyourharvs** [comment]
See https://github.com/thatguycy/WorldDynamics-Engine/issues/36
